test/src/receiver/main.cpp

```diff
@@ -12,118 +11,136 @@
-using debug=cadmium::logger::logger<cadmium::logger::logger_debug, cadmium::dynamic::logger::formatter<TIME>, oss_sink_provider>;
-using state=cadmium::logger::logger<cadmium::logger::logger_state, cadmium::dynamic::logger::formatter<TIME>, oss_sink_provider>;
-using log_messages=cadmium::logger::logger<cadmium::logger::logger_messages, cadmium::dynamic::logger::formatter<TIME>, oss_sink_provider>;
-using routing=cadmium::logger::logger<cadmium::logger::logger_message_routing, cadmium::dynamic::logger::formatter<TIME>, oss_sink_provider>;
-using global_time=cadmium::logger::logger<cadmium::logger::logger_global_time, cadmium::dynamic::logger::formatter<TIME>, oss_sink_provider>;
-using local_time=cadmium::logger::logger<cadmium::logger::logger_local_time, cadmium::dynamic::logger::formatter<TIME>, oss_sink_provider>;
-using log_all=cadmium::logger::multilogger<info, debug, state, log_messages, routing, global_time, local_time>;
-
-using logger_top=cadmium::logger::multilogger<log_messages, global_time>;
-
-
-/*******************************************/
-
-
-
-/********************************************/
-/****** APPLICATION GENERATOR *******************/
-/********************************************/
-string input_data_control = "test/data/receiver_input_test.txt";
-const char * i_input_data_control = input_data_control.c_str();
-
-std::shared_ptr<cadmium::dynamic::modeling::model> generator = cadmium::dynamic::translate::make_dynamic_atomic_model<ApplicationGen, TIME, const char* >("generator" , std::move(i_input_data_control));
-
-/********************************************/
-/****** RECIEVER *******************/
-/********************************************/
-
-std::shared_ptr<cadmium::dynamic::modeling::model> receiver1 = cadmium::dynamic::translate::make_dynamic_atomic_model<Receiver, TIME>("receiver1");
-
-
-/************************/
-/*******TOP MODEL********/
-/************************/
-cadmium::dynamic::modeling::Ports iports_TOP = {};
-cadmium::dynamic::modeling::Ports oports_TOP = {typeid(outp)};
-cadmium::dynamic::modeling::Models submodels_TOP = {generator, receiver1};
-cadmium::dynamic::modeling::EICs eics_TOP = {};
-cadmium::dynamic::modeling::EOCs eocs_TOP = {
-  cadmium::dynamic::translate::make_EOC<Receiver_defs::out,outp>("receiver1")
-};
-cadmium::dynamic::modeling::ICs ics_TOP = {
-  cadmium::dynamic::translate::make_IC<iestream_input_defs<Message_t>::out,Receiver_defs::in>("generator","receiver1")
-};
-std::shared_ptr<cadmium::dynamic::modeling::coupled<TIME>> TOP = std::make_shared<cadmium::dynamic::modeling::coupled<TIME>>(
- "TOP", 
- submodels_TOP, 
- iports_TOP, 
- oports_TOP, 
- eics_TOP, 
- eocs_TOP, 
- ics_TOP 
-  );
-
-///****************////
-
-    auto elapsed1 = std::chrono::duration_cast<std::chrono::duration<double, std::ratio<1>>>(hclock::now() - start).count();
+    using info = cadmium::logger::logger <cadmium::logger::logger_info,
```

Comment:
alignment is not per good coding practices in all using statements (line 56 to 75). Looks like I'm climbing steps.

When you break a line make sure you do it like below.
eg: Line#64 (ignore dots)
    using routing = cadmium::logger::logger <cadmium::logger::logger_message_routing, 
    .......................................................................................cadmium::dynamic::logger::formatter <TIME>,
    .......................................................................................oss_sink_provider> ;